python/src/puml/generate_plantuml_diagrams_impl.py

```diff
@@ -266,19 +345,19 @@ def puml_object(architecture_file, output_directory) -> ExecutionResult:
     # TODO: implement plugin logic here
     status = ExecutionStatus.SUCCESS
     messages: list[ExecutionMessage] = []
-    error_msg = ExecutionMessage(
-        "The puml-object command for the Generate PlantUML Diagrams plugin has not been implemented yet.",
-        MessageLevel.ERROR,
+    msg = ExecutionMessage(
+        f"Wrote PUML Object Diagram(s) to {output_directory}",
+        MessageLevel.INFO,
         None,
         None,
     )
-    messages.append(error_msg)
+    messages.append(msg)
 
     return ExecutionResult(plugin_name, "puml-object", status, messages)
 
 
 def after_puml_object_generate(
-    architecture_file: str, output_directory: str, generate: Callable
+    architecture_file: str, output_directory: str, run_generate: Any
```

Comment:
Should run_generate not be a Callable?  It also is not an arg in the docstring.

Reply:
Good catch on the docstring, I will get those updated. And believe you are correct that it can be hinted as a Callable still just like if we were direct calling to `generate`

Reply:
Also updated the `run_check` parameters and added them to the `before_puml_*` docstrings.